subsys/net/lib/lwm2m/lwm2m_obj_server.c

```diff
@@ -21,20 +21,42 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #endif
 
 #define SERVER_VERSION_MAJOR 1
+#if defined(CONFIG_LWM2M_SERVER_OBJECT_VERSION_1_1)
+#define SERVER_VERSION_MINOR 1
+#define SERVER_MAX_ID		 24
+#else
 #define SERVER_VERSION_MINOR 0
+#define SERVER_MAX_ID		 9
+#endif /* defined(CONFIG_LWM2M_SERVER_OBJECT_VERSION_1_1) */
 
 /* Server resource IDs */
-#define SERVER_SHORT_SERVER_ID		0
-#define SERVER_LIFETIME_ID		1
-#define SERVER_DEFAULT_MIN_PERIOD_ID	2
-#define SERVER_DEFAULT_MAX_PERIOD_ID	3
-#define SERVER_DISABLE_ID		4
-#define SERVER_DISABLE_TIMEOUT_ID	5
-#define SERVER_STORE_NOTIFY_ID		6
-#define SERVER_TRANSPORT_BINDING_ID	7
-#define SERVER_REG_UPDATE_TRIGGER_ID	8
-
-#define SERVER_MAX_ID			9
+#define SERVER_SHORT_SERVER_ID				0
+#define SERVER_LIFETIME_ID				1
+#define SERVER_DEFAULT_MIN_PERIOD_ID			2
+#define SERVER_DEFAULT_MAX_PERIOD_ID			3
+#define SERVER_DISABLE_ID				4
+#define SERVER_DISABLE_TIMEOUT_ID			5
+#define SERVER_STORE_NOTIFY_ID				6
+#define SERVER_TRANSPORT_BINDING_ID			7
+#define SERVER_REG_UPDATE_TRIGGER_ID			8
+#if defined(CONFIG_LWM2M_SERVER_OBJECT_VERSION_1_1)
+#define SERVER_BOOTSTRAP_UPDATE_TRIGGER_ID		9
+#define SERVER_APN_LINK_ID				10
+#define SERVER_TLS_DTLS_ALERT_CODE_ID			11
+#define SERVER_LAST_BOOTSTRAPPED_ID			12
+#define SERVER_REGISTRATION_PRIORITY_ORDER_ID		13
+#define SERVER_INITIAL_REGISTRATION_DELAY_TIMER_ID	14
+#define SERVER_REGISRATION_FAILURE_BLOCK_ID		15
```

Comment:
Nitpick: typo should be `SERVER_REGISTRATION_FAILURE_BLOCK_ID`

Reply:
Ok, Thanks Jarno. I will fix this.